webpack.config.babel.js

```diff
@@ -4,30 +4,43 @@ import path from 'path';
 import pkg from './package.json';
 
 export default env => ({
+  target: 'node',
+  node: {
+    process: false,
+  },
+
```

Comment:
Why is this needed?

Reply:
It's to make ``axios`` work both on Node.js and web.
See https://github.com/axios/axios/issues/456#issuecomment-285287911.